lms/static/sass/discussion/_discussion.scss

```diff
@@ -406,108 +359,6 @@ body.discussion {
 
     .threads {
```

Comment:
Feel like we should have a quality check for stuff like this :)

(no need to remove now)